e2e_tests/tests/experiment/test_profiling.py

```diff
@@ -12,7 +12,6 @@
 
 
 @pytest.mark.e2e_gpu
-@pytest.mark.e2e_slurm_gpu
```

Comment:
This is testing a rest API, which does not care about the resource manager.

The only thing in python code that could be tested is "can you talk to a GPU", but I think there is basically zero chance that this test fails if gpu training succeeds.